spec/support/installations.rb

```diff
@@ -95,5 +95,6 @@ def fake_installation(version, packages: [])
   allow(unity).to receive(:version) { version }
   allow(unity).to receive(:root_path) { 'foo' }
   allow(unity).to receive(:packages) { packages }
+  unity.stub(:package_installed?) { |arg| packages.include?(arg) }
```

Comment:
Maybe keep using `allow` instead of the somewhat deprecated `stub`?

https://relishapp.com/rspec/rspec-mocks/v/2-14/docs/method-stubs/stub-with-substitute-implementation